test/e2e/scale_test.go

```diff
@@ -149,14 +156,14 @@ func TestScaleTo10(t *testing.T) {
 	//add test case specific name to its own logger
 	logger := logging.GetContextLogger("TestScaleTo10")
 
-	testScaleToWithin(t, logger, 10, 30*time.Second)
+	testScaleToWithin(t, logger, 10, 90*time.Second)
```

Comment:
could we just have the heuristic and automatically compute the timeout from N? Then this parameter can go away (and if we decide to change heuristic a single place will need to be modified).

Reply:
I'd rather not do that for the time being. I see where you're coming from but I feel like fiddling with a heuristic to make it work (scalably) is not really worth the added complexity and effort right now.

Reply:
It's not clear this is easily formulated.

Thanks for this change @markusthoemmes